app/config/config.yml

```diff
@@ -12,11 +12,17 @@ framework:
     form:            true
     csrf_protection: true
     validation:      { enable_annotations: true }
-    templating:      { engines: ['twig'], assets_version: 3 }
+    templating:
+        engines: ['twig']
+    assets:
+        version: 3
     default_locale:  "%locale%"
     trusted_proxies: ~
-    trusted_hosts: ['.*\.?afsy.fr$']
-    session:         ~
+    trusted_hosts: ['^(www\.)?afsy.fr$']
```

Comment:
The `.` in `.fr` should be escaped too

Reply:
indeed (doesn't really matters anyway x))